man/networkctl.xml

```diff
@@ -374,6 +374,25 @@ s - Service VLAN, m - Two-port MAC Relay (TPMR)
         </term>
         <listitem><para>Show the specified network configuration files.</para></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term>
+          <command>edit-link</command>
```

Comment:
Instead of introducing a new verb, how about to handle the arguments that are not suffixed with `.network`, `.netdev`, or `.link`?